src/interpret.rs

```diff
@@ -86,18 +93,31 @@ fn configure_docker_switches(cmd: DockerCommandBuilder, config: &FlokiConfig) ->
 }
 
 
-fn configure_working_directory(cmd: DockerCommandBuilder, config: &FlokiConfig) -> DockerCommandBuilder {
-    cmd.set_working_directory(&config.mount)
+fn configure_working_directory(
+    cmd: DockerCommandBuilder,
+    env: &Environment,
+    floki_root: &path::Path,
+    config: &FlokiConfig,
+) -> DockerCommandBuilder {
+    cmd.set_working_directory(path::PathBuf::from(&config.mount).join(
```

Comment:
Yep, that's right.

Agree on splitting this out and adding a test - if nothing else it'll provide useful documentation.

On the `unwrap`s - if we ever hit these, it represents a dev error, not anything the user can do anything about. Given that, I thought a panic may be the "right thing" to do. I don't know what's idiomatic in rust though - happy to switch to `Result` if you'd prefer.

Reply:
Actually that's good reasoning. In general I try to avoid unwraps and panics, and if needed take the chance to output the diagnostics needed to resolve it (I wonder if it'd be worth adding some backtrace function for these things). The debug logs ought to be enough to pin this kind of thing down (`floki -vvv`).

Mild preference for `Result`, but I don't think your point of view is at all unreasonable. The `Result` will propagate to the top-level error handler anyway.

Reply:
Perhaps [expect](https://doc.rust-lang.org/std/option/enum.Option.html#method.expect) - we'll still panic on a `None` value, but we can supply a custom error message.